x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/transactions/transactions_groups_alerts.spec.ts

```diff
@@ -13,20 +13,24 @@ import { RollupInterval } from '@kbn/apm-plugin/common/rollup';
 import { apm, timerange } from '@kbn/apm-synthtrace-client';
 import { AggregationType } from '@kbn/apm-plugin/common/rules/apm_rule_types';
 import { ApmRuleType } from '@kbn/rule-data-utils';
-import { waitForAlertsForRule } from '../alerts/helpers/wait_for_alerts_for_rule';
-import { FtrProviderContext } from '../../common/ftr_provider_context';
-import { createApmRule, runRuleSoon, ApmAlertFields } from '../alerts/helpers/alerting_api_helper';
-import { waitForActiveRule } from '../alerts/helpers/wait_for_active_rule';
-import { cleanupRuleAndAlertState } from '../alerts/helpers/cleanup_rule_and_alert_state';
-
+import type { ApmSynthtraceEsClient } from '@kbn/apm-synthtrace';
+import { SupertestWithRoleScope } from '../../../../services/role_scoped_supertest';
+import { waitForAlertsForRule } from '../../../../../../apm_api_integration/tests/alerts/helpers/wait_for_alerts_for_rule';
+import {
+  createApmRule,
+  runRuleSoon,
+  ApmAlertFields,
+} from '../../../../../../apm_api_integration/tests/alerts/helpers/alerting_api_helper';
+import { waitForActiveRule } from '../../../../../../apm_api_integration/tests/alerts/helpers/wait_for_active_rule';
+import { cleanupRuleAndAlertState } from '../../../../../../apm_api_integration/tests/alerts/helpers/cleanup_rule_and_alert_state';
```

Comment:
I believe it is possible to re-use existing `alertingApi` service that we already use for other DA tests https://github.com/elastic/kibana/blob/main/x-pack/test/api_integration/deployment_agnostic/services/alerting_api.ts

Reply:
If something is missing, it would be good to port those functions into [alerting_api.ts](https://github.com/elastic/kibana/blob/main/x-pack/test/api_integration/deployment_agnostic/services/alerting_api.ts)

Reply:
Adapted service to use `alerting_api`, much better now, thanks for the suggestion!